src/schema/v2/schema.ts

```diff
@@ -128,8 +128,7 @@ const rootFields = {
   cities,
   // collection: Collection,
   _do_not_use_conversation: {
-    type: Conversation.type,
-    resolve: Conversation.resolve,
+    ...Conversation,
```

Comment:
the root issue of the args problem here - I copied an example below for `_do_not_use_image` and did not include `Conversation.args` as a result.